.gitmodules

```diff
@@ -5,3 +5,6 @@
 	path = vcpkg
 	url = https://github.com/microsoft/vcpkg.git
 	fetchRecurseSubmodules = false
+[submodule "llvm-project"]

```

Comment:
can we have sub-modules go into a sub-directory like external or 3rdparty?

It's not a huge deal but would be nice.

Reply:
since it's not a huge deal and there has apparently been discussion on this. So consider this request unrequested :P

Reply:
We can consider that if we accumulate a third submodule (at which point the benefit may outweigh the more complicated directory structure, and the one-time headache of moving submodules).